lib/aws/rails/middleware/elastic_beanstalk_worker_listener.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+module Aws
+  module Rails
+    # Middleware to handle requests from the SQS Daemon present on Elastic Beanstalk worker environments.
+    class ElasticBeanstalkWorkerListener
```

Comment:
I'd argue this isn't really a Listener -  maybe `EbsSqsActiveJobMiddleware` or something like that - trying to keep the name from being too long while being descriptive.  This more specific than handling SQS Daemon requests - it only handles requests that are translatable to ActiveJob (cron or sqs).

Reply:
Yeah I didn't care for the name and was waffling on it...I like your proposed one better.